### ISSUE TITLE
[Config Support]: Setting up group message

### ISSUE THREAD

**Issue by Swedlund** [SgtBatten/HA_blueprints#269]
### Describe the problem you are having or the outcome you are trying to achieve.

I'm trying to get group message to work. What I have done so far is to add the following into my configuration.yalm file:
```
notify:
  - platform: group
    name: all_devices
    unique_id: all_devices
    services:
      - service: OnePlus_123 <---- the name which my phone has in home assistant
      - service: iphone_123
```

In the "Frigate Camera Notificatons" blueprint and  "Notification Group" -parameter I have entered the value "notify.all_devices"

Tried a few different modifications to the config but nothing seems to work. Would appreciate if anyone that done this can help out. Thx!

### Version

0.12.0.4e

### Automation config

```yaml
alias: Motion detected - Enterence
description: ""
use_blueprint:
  path: SgtBatten/Beta.yaml
  input:
    camera: camera.front_yard
    notify_device: 6363jejek6363jfjd241
    video: >-
      {{base_url}}/api/frigate{{client_id}}/notifications/{{id}}/{{camera}}/clip.mp4
    update_thumbnail: true
    icon: mdi:cctv
    zone_filter: false
    zones: []
    message: >-
      A {{ label }} {{ 'is loitering' if loitering else 'was detected' }} on the
      {{ camera_name }} camera{% if entered_zones %} in the {{ entered_zones |
      join(', ') | replace('_',' ') }}{% endif %} at
      {{event['after']['start_time']|timestamp_custom('%H:%M')}}.
    update_sub_label: false
    attachment: snapshot.jpg?bbox=1
    notify_group: notify.all_devices
```

**Comment by SgtBatten** [comment]
What is the name of the notify entity for the mobile phone though. 

By default they all start with mobile_app
 eg mobile_app_oneplus etc

**Comment by Swedlund** [comment]
I now got it to work.

Two reasons why it did not work:

- Blueprint only wants the name of the group and not notify.<name>
- Notify entity that I put in the group was wrong. "mobile_app_" must be stated before the name of the mobile user. Dashes in name are converted to underscores

I followed the code example shown in this link: https://companion.home-assistant.io/docs/notifications/notifications-basic/#sending-notifications-to-multiple-devices
